packages/react-monaco-editor/src/utilities/defaultSupportedPackages.ts

```diff
@@ -79,7 +79,7 @@ if (typesContext) {
   hooksGroup.packages.push({ packageName: '@fluentui/react-hooks', loadTypes });
   exampleDataGroup.packages.push({ packageName: '@fluentui/example-data', loadTypes });
 }
-if (typeof window === 'object' && window.location.href.indexOf('react-charting') !== -1) {
+if (typeof window === 'object' && window.location.href.indexOf('chart') !== -1) {
```

Comment:
can we add docs why are we searching for `chart` in particular ?

Reply:
Added. Thanks.